app/routines/export_users_last_login_date.rb

```diff
@@ -0,0 +1,52 @@
+class ExportUsersLastLoginDate
+  lev_routine
+
+  protected
+
+  def exec(delete_after: true)
```

Comment:
Instead of `delete_after: true`, maybe make this `write_csv: false` (or something similar, just don't give it the same name as the method). Skip generating the CSV if set to false. Then you can also remove the delete part.

Reply:
I think we want to also test the csv generation, so we add an expectation for `remove_exported_files` to get called and stuff. So, leaving it as it is to play it safe although it seems like a great refactor.